\documentclass[11pt,oneside,letterpaper]{article}

% graphicx package, useful for including eps and pdf graphics
\usepackage{graphicx}
\DeclareGraphicsExtensions{.pdf,.png,.jpg}

% basic packages
\usepackage{color}
\usepackage{parskip}
\usepackage{float}

\definecolor{red}{rgb}{0.86,0.17,0.17}
\definecolor{green}{rgb}{0.20,0.50,0.48}
\usepackage[hidelinks]{hyperref}
\hypersetup{colorlinks=true,linkcolor=black,citecolor=black,urlcolor=green}

% text layout
\usepackage{geometry}
\geometry{textwidth=15cm} % 15.25cm for single-space, 16.25cm for double-space
\geometry{textheight=22cm} % 22cm for single-space, 22.5cm for double-space

% helps to keep figures from being orphaned on a page by themselves
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}

% bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,font=small]{caption}

% review layout with double-spacing
%\usepackage{setspace}
%\doublespacing
%\captionsetup{labelfont=bf,labelsep=period,font=doublespacing}

% cite package, to clean up citations in the main text. Do not remove.
% \usepackage{cite}
\usepackage[numbers]{natbib}
%\renewcommand\citeleft{(}
%\renewcommand\citeright{)}
%\renewcommand\citeform[1]{\textsl{#1}}

% Remove brackets from numbering in list of References
\renewcommand\refname{References}
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

\usepackage{authblk}
\renewcommand\Authands{ \& }
\renewcommand\Authfont{\normalsize \bf}
\renewcommand\Affilfont{\small \normalfont}
\makeatletter
\renewcommand\AB@affilsepx{, \protect\Affilfont}
\makeatother

%%% TITLE %%%
\title{\vspace{1.0cm} \Large \bf
Nextstrain: real-time tracking of pathogen evolution
}

\author[1,*]{James Hadfield}
\author[1]{Colin Megill}
\author[1,2]{Sidney M.~Bell}
\author[1,2]{John Huddleston}
\author[1]{Barney Potter}
\author[1]{Charlton Callender}
\author[3]{Pavel Sagulenko}
\author[1,$\S$]{Trevor Bedford}
\author[3,4,5,$\S$]{Richard A.~Neher}

\affil[1]{Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA}
\affil[2]{Molecular and Cellular Biology Program, University of Washington, Seattle, WA, USA}
\affil[3]{Max Planck Institute for Developmental Biology, T\"ubingen, Germany}
\affil[4]{Biozentrum, University of Basel, Basel, Switzerland}
\affil[5]{SIB Swiss Institute of Bioinformatics, Basel, Switzerland}
\affil[*]{To whom correspondence should be addressed}
\affil[$\S$]{These authors contributed equally to this work}

\date{}

\begin{document}

\maketitle

%%% ABSTRACT %%%
\begin{abstract}

\noindent \textbf{Summary:}
Understanding the spread and evolution of pathogens is important for effective public health measures and surveillance.
Nextstrain consists of a database of viral genomes, a bioinformatics pipeline for phylodynamics analysis, and an interactive visualisation platform.
Together these present a real-time view into the evolution and spread of a range of viral pathogens of high public health importance.
The visualization integrates sequence data with other data types such as geographic information, serology, or host species.
Nextstrain compiles our current understanding into a single accessible location, publicly available for use by health professionals, epidemiologists, virologists and the public alike.
\\
\noindent \textbf{Availability and implementation:}
All code (predominantly JavaScript and Python) is freely available from \href{https://github.com/nextstrain}{github.com/nextstrain} and the web-application is available at \href{http://nextstrain.org}{nextstrain.org}.

\end{abstract}

Viral pathogens pose an ongoing and ever-present danger to global human health, with recent epidemics such as the West-African Ebola epidemic and the ongoing Zika epidemic in the Americas highlighting the risk.
The rapid evolution of these viruses allows inference of epidemic history from genomic data.
Such analyses are often done in isolation, and may lack the spatial or temporal context in which to best interpret the results, which impedes understanding \citep{pybus2013evolutionary}.
Furthermore, the results of analyses are often not made available to the public or health bodies until after publication, which may be too late to effect change in policy or understanding.
We have developed Nextstrain to visualise outbreaks in as close to real time as possible.
Whilst currently encompassing a selection of viruses, extension to non-viral pathogens is forthcoming.

The regularly updated nature and rapidity of these analyses is crucial to the monitoring and understanding of pathogen epidemiology and evolution.
Sequencing times and costs are continually dropping, with on-the-ground sequencing being deployed during the recent West African Ebola and Zika epidemics \citep{quick2016real,faria2017epidemic}.
However, this speed of sequencing must be complemented with rapid methods by which to analyse, interpret, and disseminate results.

Nextstrain consists of three components:
\textit{Fauna} is a collection of Python scripts to maintain a database representative of all available sequences and related metadata, sourced from public repositories such as NCBI, GISAID and ViPR, as well as GitHub repositories and other sources of genomic data.
\textit{Augur} is the bioinformatics pipeline, performing subsampling, alignment, phylogenetic inference, temporal dating of ancestral nodes and discrete trait geographic reconstruction, including inference of the most likely transmission events.
This leverages the maximum likelihood phylodynamic analyses implemented in TreeTime \citep{sagulenko2017treetime}, allowing a full analysis of the entire Ebola epidemic ($n$=1581 genomes) in under 2 hours on a modern laptop computer.
These scripts separate generic core functionality from a light pathogen-specific layer such that they are easily co-opted to different pathogens.
Outputs from the pipeline are uploaded online and immediately available through the visualisation software \textit{Auspice}, located at \href{http://nextstrain.org}{nextstrain.org}.
This approach is similar in concept to Nextflu \citep{neher2015nextflu} however extended and generalised to different viral pathogens.
There is a growing need for surveillance of non-influenza viruses \citep{tang2017global}, and Nextstrain is able to be extended to nearly all outbreaks with readily accessible genomic data.

\begin{figure}[t]
\centerline{\includegraphics[width=0.8\textwidth]{figure}}
\caption{Genomic epidemiology of Zika virus as of Oct 2017 (live display at \href{http://nextstrain.org/zika}{nextstrain.org/zika}). The main interface consists of three linked panels --- a phylogenetic tree, geographic transmissions, and the genetic diversity across the genome.}
\label{nextstrain}
\end{figure}


\subsection*{Joint temporal and spatial visualisation}
Conveying understanding of pathogen evolution through space and time involves filtering large amounts of data into forms that can be easily reasoned with.
Auspice employs linked views into different facets of the data, whereby changes in one view are reflected in all others.
These views allow simultaneous interrogation of phylogenetic and spatial relationships, with additional data such as genotype or serotype expressed through colourings (Figure \ref{nextstrain}).
This is coupled with an interactive time slider to see how the pathogen has evolved and spread over the course of the epidemic.
By animating the temporal dimension, a high level overview of how the entire outbreak unfolded is quickly gained.
This approach both communicates the geographical spread of the epidemic alongside the underlying genomic data that supports this geographic reconstruction.

Maximum likelihood ancestral state reconstruction of discrete traits such as country or region of isolation allows identification of probable transmission events given the sampled data, together with a marginal likelihood for each event.
Inferred transmissions are displayed as colours on the tree and each transmission is also drawn on the map.
It is important to make known the confidence of any such reconstruction, and we do this in two ways: by matching colour saturation to the confidence of that trait, and by displaying all relevant information when one hovers over the corresponding branch or isolate on the tree.

\subsection*{Monitoring of evolution and adaptation}
Nextstrain tracks and reconstructs mutations across the tree and displays this information as a bar-chart of entropy at each position in the genome, as well as showing the mutations inferred to occur on each branch by hovering over the tree.
Selecting a position in the genome with non-zero entropy reveals the distribution of the segregating variant in the phylogeny and on the map.
This allows interrogation of genetic change which may be adaptive or underlying a change in disease dynamics.

For many pathogens, the emergence and spread of gain-of-function variants is a grave concern.
For instance, China has experienced seasonal epidemics of influenza A/H7N9 over the past five years, however human to human transmission is thought to be rare, with the majority of human cases being spillover from poultry.
The threat of mutations which facilitate human to human transmission is of extreme concern, as H7N9 has a mortality rate of around $30\%$ \citep{li2014epidemiology}.
% \textit{In vitro} experiments have identified three amino acid mutations in the hemagglutinin (HA) gene which confer a switch in specificity for human-type receptors by promoting binding to human trachea epithelial cells \citep{devries2017three}.
Nextstrain allows monitoring of experimentally determined mutations, such that interventions and control measures may be implemented where appropriate.

\subsection*{A model for public sharing of data}
Nextstrain currently presents a single, continuously updated overview of both endemic viral disease (seasonal influenza, dengue) as well as emergent viral outbreaks (avian influenza, Zika, Ebola), all based upon the same underlying bioinformatics architecture.
This architecture is well positioned to respond to future outbreaks, be they viral or bacterial.

Analysis of such outbreaks relies on public sharing of data, and Nextstrain has the ability to automatically update as new sequences from a range of public databases and repositories appear.
Scientists are justifiably hesitant to cede control of their data, and we try to address these concerns by preventing access to the raw genome sequences, and by clearly indicating the source of each sequence.
Derived data, such as phylogenetic trees, metadata and screenshots may be downloded, and private metadata may be added by users by simply dragging a CSV file onto the browser.
We believe this strikes a compromise between keeping certain data private and allowing the dissemination of results important to the wider scientific community, thereby encouraging collaboration between scientists.
Genomic epidemiology has the potential to inform the public, health organisations and scientists alike, a potential realised by sharing of data in real-time rather than retrospectively \citep{croucher2015application}.

\section*{Funding}

This work was supported by the Open Science Prize to TB and RAN, by the NSF through DGE-1256082 to SMB, by the ERC through StG-260686 to RAN and by NIH R35 GM119774-01 to TB.
TB is a Pew Biomedical Scholar.

\bibliographystyle{unsrtnat}
\bibliography{whitepaper}

\end{document}
